systemd/verdaccio.service

```diff
@@ -1,12 +1,12 @@
 [Unit]
-Description=verdaccio Service
+Description=Verdaccio lightweight npm proxy registry
 
 [Service]
 Type=simple
+Restart=always
 User=verdaccio
-WorkingDirectory=/home/verdaccio
-ExecStart=/usr/bin/env verdaccio
-ExecStop=/usr/bin/bash -c "kill $(ps -ef | grep ^verdacc+ | awk {'print $2'})"
```

Comment:
Why did you remove `ExecStop`?

Reply:
Because
1) systemd automatically kills all processes spawned by service (read `man 5 systemd.service`) and
2) you command line kills _all_ processes of user `verdaccio` (or whichever user gets abbreviated by ps to `verdacc+`).
Also, in such situations use `pkill` instead of `kill $(ps | grep)` - it's much more convenient.

Reply:
Do we have some kind of package build automation? (e.g. deb or rpm) Or is anybody downstream building and packaging this for linux distributions?

In any case, I agree with @tiandrey about the comments, if daemonizing verdaccio, as a sysadmin that's where I would expect to find the configuration files and runtime state data.

Reply:
Any reason for not using `Restart=on-failure`? I haven't used verdaccio as a daemon in a normal system, but the signal handling with dumb-init seems to be ok in the docker build.

Regarding `ExecStop`, I also don't see the need for it in principle, systemd should keep track of the spawned process id.